### ISSUE TITLE
Function arguments not reporting changes.

### ISSUE THREAD

**Issue by raydrak64** [microsoft/vscode-pylint#414]
When I make a change to a function argument in one py file the other file that calls that function does not show I have too many or too few arguments after I make the change.

Correct Example:
<test1.py>
def test(arg1, arg2, arg3)
 
<test2.py>
from test1 import test
test(arg1, arg2, arg3)

If during coding, I change test1 and eliminate arg3, or add an arg4, it does not report in test2 I have too many/few arguments.

It WILL report it, only after I reload VSC.

Also please note, I found the problem when I was using the deprecated settings having pylint enabled in the settings file. It worked flawlessly. It was only after I removed the deprecated settings and installed the Pylint extension did I notice this.

**Comment by raydrak64** [comment]
Additional information...

Once I reload VSC and it reports the problem correctly. I make the change and the error never goes away. I have to reload VSC again for it to properly acknowledge the change. It seems to "remember" only the last state.

Thank you!

**Comment by karthiknadig** [comment]
@raydrak64 Can you share the logs from Output > pylint?

**Comment by karthiknadig** [comment]
@raydrak64 I think i know what the bug is, but do share the logs anyway.

Here is a temp work around:
```
"pylint.args": ["--clear-cache-post-run=y"]
```

**Comment by karthiknadig** [comment]
This for bringing this to my notice. Please update to latest version and see if that helps.

**Comment by raydrak64** [comment]
Works brilliantly. Thank you!
